pkg/apis/openebs.io/v1alpha1/cstor_pool.go

```diff
@@ -37,10 +37,18 @@ type CStorPool struct {
 
 // CStorPoolSpec is the spec listing fields for a CStorPool resource.
 type CStorPoolSpec struct {
-	Disks    DiskAttr      `json:"disks"`
+	Group    []DiskGroup      `json:"group"`
 	PoolSpec CStorPoolAttr `json:"poolSpec"`
 }
 
+type DiskGroup struct {
+	Item []CspDisk `json:"disk"`
+}
+
+type CspDisk struct {
```

Comment:
is it possible to avoid prefix `Csp` as this struct is already within cstor_pool.go file? Does it clash with similarly named struct?

Reply:
You are right, clash is there.